lib/server.js

```diff
@@ -124,47 +140,28 @@ class Server extends KarmaEventEmitter {
     const singleRunBrowsers = new BrowserCollection(new EventEmitter())
     let singleRunBrowserNotCaptured = false
 
-    webServer.on('error', (e) => {
-      if (e.code === 'EADDRINUSE') {
-        this.log.warn('Port %d in use', config.port)
-        config.port++
-        webServer.listen(config.port, config.listenAddress)
-      } else {
-        throw e
-      }
-    })
+    webServer.on('error', (e) => { throw e })
```

Comment:
As a reader I have to ponder what will happen here.  Maybe we can have a `dieOnError(err)` function that  calls log4js, console.error and the process exit dance, then use it here and the compile fail.

Reply:
I definitely agree, but I'm not sure this pull request is right place to do it. It may require more investigation.